config/cmd/cmd.go

```diff
@@ -269,6 +275,42 @@ var (
 			EnvVars: []string{"MICRO_AUTH_EXCLUDE"},
 			Usage:   "Comma-separated list of endpoints excluded from authentication, e.g. Users.ListUsers",
 		},
+		&cli.StringFlag{
+			Name:    "auth_login_url",
+			EnvVars: []string{"MICRO_AUTH_LOGIN_URL"},
```

Comment:
Considering this is micro web specific this should live in micro/micro repo

Reply:
Good point, i'll move this over,